### PR TITLE
Update abap_regex,to_mixed.

### PR DESCRIPTION
**Pull request by Dwivedirahul44**
Update Punctuation class members

### CODE REVIEW

**Comment by Dwivedirahul44** [comment]
@larshp - please review and add suggestions or merge.

File: packages/runtime/src/builtin/to_mixed.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
 /* eslint-disable @typescript-eslint/ban-types */
 import {String} from "../types";
 import {ICharacter} from "../types/_character";
-import {INumeric} from "../types/_numeric";
 
 export function to_mixed(input: {
   val: ICharacter | string,
   sep?: ICharacter | string,
-  case?: ICharacter | string,
-  min?: INumeric | number }): String {
```
Comment (larshp) [inline packages/runtime/src/builtin/to_mixed.ts:10]: please keep `min` in the code, I know its not used at the moment, but it is a parameter of the method

**Review by larshp — CHANGES_REQUESTED** [review]

File: packages/runtime/src/abap_regex.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ export class ABAPRegExp {
   // converts from ABAP specific regex to javascript regex
   public static convert(input: string): string {
     let ret = input;
-    ret = input.replace(/\[\[:punct:\]\]/g, "[\\.\\,]");
+    ret = input.replace(/\[\[:punct:\]\]/g, "[\\.\\,\\-\\{\\}\\[\\]\\:\\!\\?\\(\\)\\;\\']");
```
Comment (Dwivedirahul44) [inline packages/runtime/src/abap_regex.ts:6]: update class members of punct

File: test/builtin/replace.ts [hunk 3]
```diff
@@ -103,13 +103,13 @@ write text.`;
     const code = `
 DATA plain_text TYPE string.
 DATA result TYPE string.
-plain_text = |hello, world.moo|.
+plain_text = |H-\\{\\}[...]ello, Wo:,rld!?.();'|.
```
Comment (Dwivedirahul44) [inline test/builtin/replace.ts:106]: update test case

**Comment by larshp** [comment]
thanks 👍 

will be published in next version

**Pull request merged**